server/routes/allRecipeRoute.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+
+const router = express.Router();
+const passport = require('passport');
+// eslint-disable-next-line no-unused-vars
+const passportConfig = require('../passport');
```

Comment:
we talked about this in the other PR. I asked you why you need this and you said

> eslint would not allow me to maintain a consistent "err"/ "error" naming variable as arguments throughout the file.

But "no-unused-vars" should just be about declaring variables you aren't using, which in this case is passportConfig. Why do you need passportConfig here again?

Reply:
unfortunately i dont remember the reason why this was added, i believe there was an error before regarding to information leak; however, after removing this to check to errors, errors don't seem to persist. Thus, i am removing this line, until error shows again.